tests/unit/t_version.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <aws/cryptosdk/version.h>
+#include "testing.h"
+
+static int print_version_info() {
+    printf(
```

Comment:
This test isn't really testing anything automatically...

Reply:
It wasn't meant to be. It was just for the convenience of a developer being able to type ``tests/unit-test-suite version`` at the command line and see what all the strings were. I can take it out if you feel strongly.

Reply:
Removed.

Reply:
@ttjsu-aws regarding your comment about the unit test, I don't think there's actually meaningful functionality here that a unit test would be helpful for. I just created this here as a substitute for having a command line tool. But I got rid of it per @bdonlan's comment, and it's easy enough to re-add later if we decide on a good place for it.